addons/mail/static/src/core/common/composer_actions.js

```diff
@@ -104,8 +104,9 @@ composerActionsRegistry
         condition: (component) => {
             const thread = component.thread ?? component.message?.thread;
             return !(
-                thread?.channel_type === "whatsapp" &&
-                component.props.composer.attachments.length > 0
+                (thread?.channel_type === "whatsapp" &&
+                    component.props.composer.attachments.length > 0) ||
+                component.props.composer.portalComment
```

Comment:
It might be easier to read with `&& !component.props.composer.portalComment` outside of the main "not".

Maybe it's personal preference but I always find it easier to read when conditions are distributed.